app/controllers/v0/dependents_applications_controller.rb

```diff
@@ -2,6 +2,9 @@
 
 module V0
   class DependentsApplicationsController < ApplicationController
+    include Traceable
```

Comment:
Is the goal to add this to all controllers? I'm curious if it would make sense at all to move the `include Traceable` to the `ApplicationController` class with a default `service_tag` and then in controllers that inherit from `ApplicationController` to just override the `service_tag`. Also, this is on a different tangent, but how do you deal tag with controllers that are associated with multiple services?

Reply:
Good call out, I realized in a subsequent PR that we can tweak the Traceable concern slightly to make it inheritable in subclasses of Application Controller, so I'll probably re-work this PR slightly accordingly.